app.go

```diff
@@ -25,6 +25,7 @@ type App struct {
 	cursorColor       Color
 	music             *Music
 	sound             *Sound
+	egoSelected       string
```

Comment:
Why don't we do this instead?

```suggestion
	ego       *Actor
```

It is true that if we do that, we should have a function to move an actor by its pointer. But we might apply that change. Adding a `walkToPosition()` method to `Actor` type and invoke it from the input event controller, and also from the command execution handler.

Reply:
Done!